docs/source/guide/index.md

```diff
@@ -53,21 +53,218 @@ You're ready to start [labeling and annotating your data](labeling.html)!
-| Templates | Example labeling configurations that you can use to specify the type of labeling that you're performing with your dataset. See [all available templates](/templates) |
-| Tags | Configuration options to customize the labeling interface. See [more about tags](/tags). |
+When you upload data to Label Studio, each item in the dataset becomes a labeling task. The following table defines the terms that appear throughout Label Studio.
+
+**Annotation**
+Annotation is the process of labeling data in various formats such as video, images, or text, so that machines can understand it. The output of a labeling task which was previously called **Completions**. 
+
+**Artificial Intelligence (AI)**
+AI is the intelligence demonstrated by machines, as opposed to the natural intelligence displayed by animals including humans.  It refers to the simulation of human intelligence in machines that are programmed to think like humans and mimic their actions.
+
+**Audio classification**
+Audio classification or sound classification can be referred to as the process of analyzing audio recordings. This amazing technique has multiple applications in the field of AI and data science such as chatbots, automated voice translators, virtual assistants, music genre identification, and text to speech applications.
+
+**Audio labeling**
+Audio labeling or speech labeling is a standard annotation technique that concerns separating sounds and labeling with specific metadata.
+
+**Audio tagging**
+Audio tagging is a task to predict the tags of audio clips. This includes music tagging, acoustic scene classification, audio event classification, and so on.
+
+**Audio transcription**
+Audio transcription transforms any video or audio file into an entire experience for your audience. It is available in a written format of any verbal communication.
+
+**Automatic Speech Recognition (ASR)**
+ASR refers to the problem of getting a program to automatically transcribe spoken language (speech-to-text).
+
+**Bounding box object detection**
+A bounding box describes the spatial location of an object. It is rectangular, which is determined by the "x" and "y" coordinates of the upper-left corner of the rectangle and such coordinates of the lower-right corner.
+
+**Consolidated Receipt Dataset for Post-OCR Parsing (CORD)**
+COResets and Data Subset slection is an efficient and scalable library for making machine learning time, energy, cost, and compute efficient built on top of pytorch. It uses a subset of training data for model training and incorporates various state-of-the-art data selection strategies to select the proper representative data subset from massive datasets. 
+
+**Data annotation**
+Data annotation is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.
```

Comment:
[Data annotation](https://labelstud.io/blog/from-raw-data-to-a-trained-model-automate-your-ml-pipeline-with-label-studio-and-amazon-sagemaker/) is the categorization and labeling of data for AI applications. It helps machines to learn the certain patterns and correlate the results, and the use the data sets to recognize the similar patterns in future to predict the results.